tests/test_provisioners.py

```diff
@@ -125,6 +126,11 @@ def test_cluster_start(mocker, cluster_provisioner, ssh_key):
         'Name': ANY,
         'ReleaseLabel': 'emr-%s' % emr_release,
         'ServiceRole': 'EMR_DefaultRole',
+        'Steps': [{'ActionOnFailure': 'TERMINATE_JOB_FLOW',
+                   'HadoopJarStep': {'Args': ['s3://telemetry-spark-emr-2/steps/zeppelin/zeppelin.sh'],
+                                     'Jar': 's3://us-west-2.elasticmapreduce/libs/script-runner/script-runner.jar'},
```

Comment:
No need to actual test the URL here, we're just looking to confirm that the general argument signature is what we expect. So just use `cluster_provisioner.jar_uri`  and cluster_provisioner.zeppelin_uri here.

Also, please indent the list of steps a tiny bit similar to the other tests, e.g. see `test_spark_job_run`. Thanks 😍